sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/HttpPipelineHelper.cs

```diff
@@ -51,6 +51,23 @@ internal static int GetRetryInterval(Response httpResponse)
             return MinimumRetryInterval;
         }
 
+        internal static TimeSpan GetRetryIntervalTimespan(Response httpResponse)
+        {
+            if (httpResponse != null && httpResponse.Headers.TryGetValue(RetryAfterHeaderName, out var retryAfterValue))
+            {
+                if (int.TryParse(retryAfterValue, out var delaySeconds))
+                {
+                    return TimeSpan.FromSeconds(delaySeconds);
+                }
+                if (DateTimeOffset.TryParse(retryAfterValue, out DateTimeOffset delayTime))
+                {
+                    return (delayTime - DateTimeOffset.Now);
+                }
+            }
+
+            return TimeSpan.MinValue;
+        }
```

Comment:
Consider changing this to a TryGet method.
My reasoning is that the MinValue return value is never used, instead it's compared in a ternary. 
In my mind, you can return a bool and save the extra comparison.

```
var retryAfterInterval = HttpPipelineHelper.GetRetryIntervalTimespan(response);
var backOffTimeInterval = retryAfterInterval != TimeSpan.MinValue ? retryAfterInterval : GetBackOffTimeInterval();
```

could be rewritten as:

```
var backOffTimeInterval = HttpPipelineHelper.TryGetRetryIntervalTimespan(response, out var retryAfterInterval )
    ? retryAfterInterval 
    : GetBackOffTimeInterval()
```